hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -730,18 +731,18 @@ private KeyInfo getProtobuf(boolean ignorePipeline, String fullKeyName,
     return kb.build();
   }
 
-  public static OmKeyInfo getFromProtobuf(KeyInfo keyInfo) {
+  public static Builder newBuilderFromProtobufPartial(KeyInfo keyInfo) {
```

Comment:
Change it similarly as `OmBucketInfo`, i.e. use one method to include/exclude large fields.

Reply:
Ok, done.